pkg/kdk/pull.go

```diff
@@ -46,12 +58,16 @@ func pullImage(cfg *KdkEnvConfig, imageCoordinates string) error {
 	}
 	defer out.Close()
 
-	// Silence the output for Info loglevel, but allow it through all
-	//   others.  This is a cheap way to enable streaming output.
-	if log.GetLevel() == log.InfoLevel {
-		io.Copy(ioutil.Discard, out)
-	} else {
-		io.Copy(os.Stdout, out)
+	var message ProgressMessage
+	scanner := bufio.NewScanner(out)
+	for scanner.Scan() {
+		b := scanner.Bytes()
+		json.Unmarshal(b, &message)
+		fmt.Printf("%+v\n", message)
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Fatal(err)
```

Comment:
SHOULD:

log.WithField("err", error).Fatal("Failed to read output from docker client pull")

Reply:
Fixed. Thanks